locales/templates/app.pot

```diff
@@ -219,11 +231,8 @@ msgstr ""
 #. content to show up in the "recent apps" screen      and prevents the user
 #. from taking screenshots or casting the app to other displays. %1$s will be
 #. replaced with the      name of the app (e.g. Firefox Focus).
-#, c-format
 msgctxt "preference_privacy_secure_mode_summary"
-msgid ""
-"Turn off stealth mode to take screenshots and see sites you’ve visited "
-"using %1$s in your Recent Apps list"
+msgid "Hide webpages when switching apps"
```

Comment:
This doesn't look right. The string completely change, but the ID and comment remained the same?

The comment is completely off at this point.